server.js

```diff
@@ -5,24 +5,23 @@ const express = require("express");
 const cors = require("cors");
 const Book = require("./models/bookModel");
 const app = express();
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 
 app.get("/test", (request, response) => {
   response.send("test request received");
 });
 
 app.use(cors());
-app.use(express.json())
+app.use(express.json());
 
-app.get('/test', (request, response) => {
-
-  response.send('test request received')
-
-})
+app.get("/test", (request, response) => {
+  response.send("test request received");
+});
 
-app.get('/books', handleGetBooks)
-app.post('/books:id',handlePostBooks)
-app.delete('/books:id', handleDeleteBooks)
+app.get("/books", handleGetBooks);
+app.post("/books:id", handlePostBooks);
+app.delete("/books:id", handleDeleteBooks);
```

Comment:
Good work on edits for delete and create 👍 

- line 23 is missing '/` should read: `/books/:id'`